README.md

```diff
@@ -12,7 +12,7 @@ Requesting results from your back-end can be useful in order to control result r
 ## 1. Install
 
 ```
-pip install constructor-io
+pip install constructorio-python
```

Comment:
We changed this on master but it didn't get changed here. Can we manually change this to:
`pip install constructor-io`